scripts/deploy_docker_image.sh

```diff
@@ -0,0 +1,3 @@
+docker pull ghcr.io/lizlooney/object_detection:2021_11_25
+docker tag ghcr.io/lizlooney/object_detection:2021_11_25 gcr.io/${FMLTC_GCLOUD_PROJECT_ID}/object_detection:2021_11_25
```

Comment:
This probably needs a comment stating that it won't work for people who clone and self-host.

Reply:
This will work fine for people who clone and self-host.